tiledb/cloud/notebook.py

```diff
@@ -35,3 +54,288 @@ def rename_notebook(
         )
     except GenApiException as exc:
         raise tiledb_cloud_error.check_exc(exc) from None
+
+
+def download_ipnyb_file_name_from_cloud(
+    tiledb_uri: str,
+    storage_credential_name: str,
+    ipynb_file_name: str,
+    *,
+    character_encoding: Optional[str] = 'utf-8',
+) -> None:
+    """
+    Downloads a notebook file from TileDB Cloud to local disk.
+    :param str tiledb_uri: such as "TileDB-Inc/quickstart_dense".
+    :param str storage_credential_name: such as "janedoe-creds", typically from the user's account settings.
```

Comment:
I am open to correction from other reviewers but, for easier reading as a user, I prefer to wrap the docstrings at 80 cols and continue the descriptions on the next line with a two-space indent:

```
    :param storage_credential_name: yada yada yada
      yada yada
```